core/Array.carp

```diff
@@ -240,4 +240,18 @@
     (zip &Pair.init-from-refs
          &(range 0 (length xs) 1) ;; Inefficient, creates a temporary array.
          xs))
+
+  (doc remove "Remove occurrences of element el in array arr, in-place")
+  (defn remove [el arr]
+    (filter &(fn [x] (not (= el x)))
+            arr))
+
+  (doc remove-idx "Remove element at index idx from arr")
+  (defn remove-idx [i arr]
+    (do
+      ;;(assert (<= 0 i))
+      ;;(assert (< i (Array.length &arr)))
```

Comment:
I thought about adding assert as a template, but I don't want a separate wrapper function (line numbers will be misleading, for instance)

Reply:
How about naming this `remove-nth`? That name seems a little more congruent with the rest of the API?

Reply:
definitely